src/Factory.php

```diff
@@ -141,13 +141,25 @@ public function target($name)
             if (preg_match('#^https?://#', $file)) {
                 $files[] = new Remote($file);
             } else {
-                $path = $scanner->find($file);
-                if ($path === false) {
-                    throw new RuntimeException("Could not locate $file for $name in any configured path.");
+                if (preg_match('/(.*)(\/\*.*?)$/', $file, $matches)) {
```

Comment:
Yeah, having a separate case will look much better. The case would also check that the target dir exists, otherwise throw an exception (like done in Local case).
What do you think about supporting recursion as well? For example, using a format with double wildcards like ```/path/**``` or ```/path/**/*.js``` (could be useful for large apps with several directory levels).